### PR TITLE
V3/fix/remote cozy

### PR DESCRIPTION
**Pull request by sebn**
Various little details that were not spotted during reviews of the `RemoteCozy` class (including mine).

### CODE REVIEW

File: test/unit/remote/cozy.js [hunk 1]
```diff
@@ -56,13 +56,13 @@ describe('RemoteCozy', function () {
...
-          let dir = await builders.dir(remoteCozy).build()
-          let file = await builders.file(remoteCozy).inDir(dir).build()
+          let dir = await builders.dir().build()
+          let file = await builders.file().inDir(dir).build()
 
           let newChanges = await remoteCozy.changes(seq)
           let ids = newChanges.results.map(result => result.id).sort()
 
-          ids.should.eql([file._id, dir._id].sort())
+          ids.sort().should.eql([file._id, dir._id].sort())
```
Comment (enguerran) [inline test/unit/remote/cozy.js]: On L63, you set `ids` to `something.sort()`.
Then you `sort()` it again for the equality check.

Is it a mislead?

**Comment by sebn** [comment]
Useless commit removed. CI is green.

File: test/unit/remote/cozy.js (hunk 1 above)
Comment (sebn) [inline test/unit/remote/cozy.js]: Removed commit

**Comment by sebn** [comment]
:ship: @kosssi @enguerran ?

**Comment by enguerran** [comment]
:shipit:

**Pull request merged**